Gemfile

```diff
@@ -3,10 +3,11 @@ source 'https://rubygems.org'
 gem "jekyll", '>= 4.2.0'
 gem 'json', '>= 2.3.0'
 gem 'jekyll-redirect-from', '>= 0.15.0'
-gem 'jekyll-sitemap', '>= 1.3.1'
+gem 'jekyll-sitemap', '>= 1.4.0'
 gem 'scss_lint'
 gem 'jekyll-include-cache'
 gem 'liquid-c'
+gem 'html-proofer'
```

Comment:
Install the `html-proofer` gem to allow link and HTML validation proofing.
`html-proofer`: https://rubygems.org/gems/html-proofer/versions/3.4.0